src/SprykerEco/Zed/ProductManagementAi/Business/ProductManagementAiFacadeInterface.php

```diff
@@ -35,9 +35,9 @@ public function proposeCategorySuggestions(string $productName, string $descript
      * @param string $imageUrl
      * @param string $targetLocale
      *
-     * @return \Generated\Shared\Transfer\OpenAiChatResponseTransfer
+     * @return \Generated\Shared\Transfer\PromptResponseTransfer
      */
-    public function generateImageAltText(string $imageUrl, string $targetLocale): OpenAiChatResponseTransfer;
+    public function generateImageAltText(string $imageUrl, string $targetLocale): PromptResponseTransfer;
```

Comment:
please unify facade input/output arguments or register ticket for this, as for now it's not extendable (aka I have 2 image URls or I need alt for 2 locales ... same for all methods in this facade)